HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -244,6 +244,13 @@ def checkHLTfor43774(process):
 
     return process
     
+def customizeHLTFor44576(process):
```

Comment:
```suggestion
def customizeHLTfor44576(process):
```

@emiglior @bardellig please test the branch before submitting more updates.